i18n/venue/index.i18n.js

```diff
@@ -143,8 +120,8 @@ export default genI18nMessages({
                 'PyCon Taiwan 2023 將在中央研究院人文社會科學館舉辦，聚焦於 Python 的最新技術和最佳實踐，讓開源社群帶來高品質的演講和工作坊，無論您是 Python 的新手還是資深開發人員，都能在這裡找到有價值的學習和交流機會。',
```

Comment:
Same here.

Reply:
This would be updated at https://github.com/pycontw/pycontw-frontend/pull/533/commits/23da2ac5b2d14db4f9d0850f5227fc87cf190923
plz check.

Reply:
LGTM